src/MAUI/Maui.Samples/Samples/Layers/AddVectorTiledLayerFromCustomStyle/AddVectorTiledLayerFromCustomStyle.xaml.cs

```diff
@@ -0,0 +1,165 @@
+
+        // Path to Dodge City vector tile package.
+        private readonly string _localVectorPackagePath = DataManager.GetDataFolder("f4b742a57af344988b02227e2824ca5f", "dodge_city.vtpk");
+
+        private List<PortalItem> _vectorTiledLayers = new List<PortalItem>();
+
+        private readonly Viewpoint _defaultViewpoint = new Viewpoint(10, 5.5, 1e8);
+        private readonly Viewpoint _dodgeCityViewpoint = new Viewpoint(37.76528, -100.01766, 4e4);
+
+        private ItemResourceCache _lightStyleResourceCache;
+        private ItemResourceCache _darkStyleResourceCache;
+
+        public AddVectorTiledLayerFromCustomStyle()
+        {
+            InitializeComponent();
+            _ = Initialize();
+        }
+
+        private async Task Initialize()
+        {
+            try
+            {
+                // Load the default portal.
+                ArcGISPortal portal = await ArcGISPortal.CreateAsync();
+
+                // Store a list of all portal items.
+                foreach (string item in _onlineItemIDs)
+                {
+                    PortalItem portalItem = await PortalItem.CreateAsync(portal, item);
+                    _vectorTiledLayers.Add(portalItem);
+                }
+                foreach (string item in _offlineItemIDs)
+                {
+                    PortalItem portalItem = await PortalItem.CreateAsync(portal, item);
+                    _vectorTiledLayers.Add(portalItem);
+                }
+
+                // Create a map using defaults.
+                MyMapView.Map = new Map(new Basemap(new ArcGISVectorTiledLayer(_vectorTiledLayers[0]))) { InitialViewpoint = _defaultViewpoint };
+
+                // By default, the UI label will not reflect the default style.
+                ChosenStyle.Text = "Default";
+
+                // Export offline custom styles.
+                _lightStyleResourceCache = await ExportStyle(_vectorTiledLayers[4]);
+                _darkStyleResourceCache = await ExportStyle(_vectorTiledLayers[5]);
+            }
+            catch (Exception ex)
+            {
+                // Report exceptions.
+                await Application.Current.MainPage.DisplayAlert("Error", ex.Message, "OK");
+            }
+        }
+
+        private void ButtonClick(object sender, EventArgs e)
+        {
+            _ = StyleChooser();
+        }
+
+        private async Task StyleChooser()
+        {
+            try
+            {
+                // Get the chosen style, determine that style's index, and update UI label.
+                string style = await Application.Current.MainPage.DisplayActionSheet("Select:", "Cancel", null, styles);
+                // - Don't attempt to change the style if the user cancels or misclicks.
+                if (style is null || style.Equals("Cancel")) { return; }
+                int styleIndex = Array.IndexOf(styles, style);
+                ChosenStyle.Text = style;
```

Comment:
```suggestion
                ChosenStyle.Text = $"Current style: {style}";
```
I think this UI pattern is slightly more descriptive for the user.